src/arch/xtensa/CMakeLists.txt

```diff
@@ -328,12 +334,21 @@ if(MEU_PATH)
 		USES_TERMINAL
 	)
 else()
+	if(NOT DEFINED ENV{SOF_RIMAGE_KEY})
```

Comment:
Please don't use env variables. I know they may look handy but it makes harder to figure out what comes from where. Also other params are not settable this way, so if we decide to support env variables we would have to do it for all variables for consistency.